examples/kubernetes/gateway/basic-https.yaml

```diff
@@ -55,5 +55,5 @@ spec:
         type: PathPrefix
         value: /
     backendRefs:
-    - name: productpage
-      port: 9080
+    - name: frontend
+      port: 80
```

Comment:
My original intention is do use productpage service instead of frontend service. 

Any benefit to change to frontend?